fiona/ogrext.pyx

```diff
@@ -255,18 +255,26 @@ cdef class FeatureBuilder:
         cdef void *cogr_geometry = NULL
 
         if not ignore_geometry:
-            cogr_geometry = OGR_F_GetGeometryRef(feature)
+            """ We steal the geometry: the geometry of the in-memory feature is now null
+            and we are responsible for cogr_geometry.
+            """
```

Comment:
@rbuffat I now see that stealing the geometry is the proper thing to do. Please convert the comment to use `#` instead of triple quotes.